src/core/common/instance.cpp

```diff
@@ -167,6 +167,14 @@ void Instance::Reset(void)
     otPlatReset(this);
 }
 
+#if OPENTHREAD_RADIO
+void Instance::RadioStackReset(void)
+{
+    mRadio.Reset();
+    mLinkRaw.Reset();
```

Comment:
need to call SubMac reset here?

Reply:
Updated. `SubMac::Init` is called from `LinkRaw::Init`.